beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md

```diff
@@ -80,6 +80,7 @@ Set the `LAUNCH_ENVIRONMENT_FILE_ID` to the copied Environment File ID in the `A
 ![](../../../.gitbook/assets/edge_analytics_report.png)
```

Comment:
We need to indent here.

Reply:
The image shows very small in gitbook when indented :(  Can we try this to see if it works or can we use bullet points?